docroot/themes/humsci/su_humsci_gin_admin/su_humsci_gin_admin.theme

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Why not put this same stuff into the `hs_admin` that is part of this PR. One place to diagnose admin issues is preferred.

Reply:
Fixed. Moved the code to `hs_admin` module and removed this file.